pcdsdevices/slits.py

```diff
@@ -492,3 +500,49 @@ class PowerSlits(BeckhoffSlits):
 
     rtds = DDCpt(_rtd_fields(RTD, 'rtd', range(1, 9)))
     fsw = Cpt(NotImplementedSignal, ':FSW', kind='normal')
+
+
+
+class ExitSlits(BaseInterface,Device,LightpathInOutMixin):
+ 
+    tab_component_names = True
+
+    lightpath_cpts = ['target']
+    _icon = 'fa.video-camera'
+
+    target = Cpt(TwinCATStatePMPS, ':YAG:STATE', kind='hinted',
+                 doc='Control of the YAG  stack via saved positions.')
+    yag_motor = Cpt(BeckhoffAxis, ':MMS:YAG', kind='normal',
+                  doc='Direct control of the Yag Stack motor.')
```

Comment:
Similar to the other PR, make sure lines like this (`doc=...` here) align with the character after `(` on the previous line.
For example:
```python
    yag_motor = Cpt(BeckhoffAxis, ':MMS:YAG', kind='normal',
                    doc='Direct control of the Yag Stack motor.')
```